common/src/main/java/com/microsoft/identity/common/internal/controllers/NativeAuthController.kt

```diff
@@ -0,0 +1,92 @@
+import com.microsoft.identity.common.internal.telemetry.events.ApiEndEvent
+import com.microsoft.identity.common.java.AuthenticationConstants
+import com.microsoft.identity.common.java.cache.ICacheRecord
+import com.microsoft.identity.common.java.commands.parameters.SilentTokenCommandParameters
+import com.microsoft.identity.common.java.commands.parameters.nativeauth.AcquireTokenNoFixedScopesCommandParameters
+import com.microsoft.identity.common.java.commands.parameters.nativeauth.BaseNativeAuthCommandParameters
+import com.microsoft.identity.common.java.configuration.LibraryConfiguration
+import com.microsoft.identity.common.java.controllers.CommandDispatcher
+import com.microsoft.identity.common.java.controllers.results.ICommandResult
+import com.microsoft.identity.common.java.dto.AccountRecord
+import com.microsoft.identity.common.java.eststelemetry.PublicApiId
+import com.microsoft.identity.common.java.exception.ArgumentException
+import com.microsoft.identity.common.java.exception.ClientException
+import com.microsoft.identity.common.java.exception.ErrorStrings
+import com.microsoft.identity.common.java.exception.ServiceException
+import com.microsoft.identity.common.java.logging.LogSession
+import com.microsoft.identity.common.java.logging.Logger
+import com.microsoft.identity.common.java.providers.microsoft.microsoftsts.MicrosoftStsAuthorizationRequest
+import com.microsoft.identity.common.java.providers.microsoft.microsoftsts.MicrosoftStsOAuth2Strategy
+import com.microsoft.identity.common.java.providers.nativeauth.NativeAuthOAuth2Strategy
+import com.microsoft.identity.common.java.providers.nativeauth.responses.ApiErrorResult
+import com.microsoft.identity.common.java.providers.oauth2.OAuth2Strategy
+import com.microsoft.identity.common.java.providers.oauth2.OAuth2StrategyParameters
+import com.microsoft.identity.common.java.providers.oauth2.OAuth2TokenCache
+import com.microsoft.identity.common.java.request.SdkType
+import com.microsoft.identity.common.java.result.AcquireTokenResult
+import com.microsoft.identity.common.java.result.LocalAuthenticationResult
+import com.microsoft.identity.common.java.telemetry.TelemetryEventStrings
+import com.microsoft.identity.common.java.util.StringUtil
+import com.microsoft.identity.common.java.util.ThreadUtils
+import lombok.EqualsAndHashCode
+import java.io.IOException
+import java.net.URL
+
+/**
+ * The implementation of MSAL Controller for Native Authentication.
+ */
+@EqualsAndHashCode(callSuper = true, onlyExplicitlyIncluded = true)
+class NativeAuthController : BaseNativeAuthController() {
+
+    companion object {
+        private val TAG = NativeAuthController::class.java.simpleName
+    }
+    
+    fun acquireTokenSilent(
+        parameters: AcquireTokenNoFixedScopesCommandParameters
+    ): AcquireTokenResult {
+        LogSession.logMethodCall(TAG, "${TAG}.acquireTokenSilent")
+
+        val acquireTokenSilentResult = AcquireTokenResult()
```

Comment:
This just seems an empty result returning without doing any real operation. Perhaps we need TODOs here to add real implementation?